### PR TITLE
issue-5039: added documentation for curve attribute on EllipticCurveP…

### PR DESCRIPTION
**Pull request by jschmidtlein**
…rivateKey based on existing docs

### CODE REVIEW

File: docs/hazmat/primitives/asymmetric/ec.rst [hunk 1]
```diff
@@ -611,6 +611,12 @@ Key Interfaces
             :func:`~cryptography.hazmat.primitives.asymmetric.utils.decode_dss_signature`,
             which returns the decoded tuple ``(r, s)``.
 
+    .. attribute:: curve
+
+            :type: :class:`EllipticCurve`
```
Comment (alex) [inline docs/hazmat/primitives/asymmetric/ec.rst]: This should all be intended one amount less.

Comment (jschmidtlein) [reply docs/hazmat/primitives/asymmetric/ec.rst]: Ah! Sorry about the delay on fix. Taken care of.

**Comment by alex** [comment]
Can you quickly rebase/merge master here? The failing build is one we just removed.

**Comment by jschmidtlein** [comment]
My git-fu is pretty weak, but I think I got it done. Failing that, I'll recheck tomorrow.

**Pull request merged**